### ISSUE TITLE
Additional element in coar_publication_types.xsd e.g.  COAR resource type c_1843 

### ISSUE THREAD

**Issue by ACz-UniBi** [openaire/guidelines-cris-managers#128]
In some CRIS systems, we notice that the resourceType _Other_ (http://purl.org/coar/resource_type/c_1843) is used in `<oai_cerif:Publication>` elements as `<oai_cerif:Type xmlns:oai_cerif="https://www.openaire.eu/cerif-profile/vocab/COAR_Publication_Types">` .

This resource type isn't mentioned in the file _schemas/vocabularies/coar_publication_types.xsd_ . 
It's contained in the xsd files at _schemas/vocabularies/_
* coar_product_types.xsd
* coar_resourcetypes.xsd

Should this also be available in _coar_publication_types.xsd_ and further schema-files?

**Comment by jdvorak001** [comment]
Yes, "other" should be allowed in all of the three distilled schemes. I'll try to add it in the coar_vocab_v3 branch.

**Comment by alubie** [comment]
However, applying the resource type other should be discouraged by the guidelines - if a more specific resource type is appropriate.

**Comment by jdvorak001** [comment]
(in a call with @ACz-UniBi) On a second look, this may not be necessary. 

For publications, **other periodical** (http://purl.org/coar/resource_type/QX5C-AR31 – *A resource type that is not included in existing terms under the top concept "Text"*) can serve the purpose. @ACz-UniBi to discuss the motivation with the requesting data source.

We definitely do not need **other** with Patents.

**Comment by ACz-UniBi** [comment]
Dear @olli-gold , thank you very much for your additional note.

do you look for a default "fallback" for publications? In this case, I propose to use the generic ResourceType "text"  for publications ( https://github.com/openaire/guidelines-cris-managers/blob/master/docs/cerif_xml_publication_entity.rst#type ) instead of the specific one under the hierarchy "text".

Best, Andreas

**Comment by jdvorak001** [comment]
I would second @ACz-UniBi on this. If something is a Publication, it has to be Text (http://purl.org/coar/resource_type/c_18cf) or something more specific. If you can't find the more specific term for your case, then state Text.

The term "Other periodical" (http://purl.org/coar/resource_type/QX5C-AR31) in COAR Resource Types v.3.0 is confusing me a bit. Both its definition and its position in the hierarchy seem to imply "Other text" would perhaps be a more fitting label. But then, it's a question if you really need that inside your hierarchy. I would even suggest we discourage implementers from using it, by: (a) postponing its introduction in the CRIS Guidelines, or (b) introducing it now, but marking it deprecated.
